backend/app/api/api_v1/routers/summaries.py

```diff
@@ -25,8 +26,25 @@ def search_by_country(
     request: Request,
     geography_id: int,
     current_user=Depends(get_current_active_db_user),
+    db=Depends(get_db),
 ):
-    # FIXME: Implement this RDS search
+    """Searches the documents filtering by country and grouping by category."""
+    top_documents = {}
+    document_counts = {}
+
+    for cat in CategoryName:
+        results = browse_rds(db, BrowseArgs(geography_id=geography_id, category=cat))
+        document_counts[cat] = len(results.documents)
+        top_documents[cat] = list(results.documents[:5])
+
+    events = get_events_for_country(db, geography_id)
+
+    # TODO: Add targets
+    targets = []
```

Comment:
ugh, targets... something else we need to make a plan for asap!

Reply:
lol ... I did mention that to Michal too.